js/script.js

```diff
@@ -195,22 +188,15 @@ const setUpParkingLotsLayer = (map) => {
 const setUpSite = () => {
   setUpAbout();
   const map = createMap();
-  const locationTag = extractLocationTag(window.location.href);
-  if (locationTag) {
-    $("#toggle-projects-by-tag").css("display", "none");
```

Comment:
I couldn't find this ID `toggle-projects-by-tag` anywhere in the repo. It's dead code.